src/docfx/lib/markdown/XrefExtension.cs

```diff
@@ -27,7 +27,7 @@ internal static class XrefExtension
 
                 if (xrefLink.Href is null)
                 {
-                    return new LiteralInline(raw);
+                    return new NoLocXrefContainer(new LiteralInline(raw));
```

Comment:
We could use `HtmlInline` to replace this `NoLocXrefContainer`.

Reply:
`htmlTag`="<span class=no-loc>...</span>"?

Reply:
Use nolocinline now